### ISSUE TITLE
[question/bug] Android 11 partial incompatibility

### ISSUE THREAD

**Issue by gbdomubpkm** [brarcher/video-transcoder#194]
Hello.

Will this great and very useful appl go back into development ? Or at least, will someone have the skills and the goodwill to make fully compatible android 11 at least, what already exists?

The first very annoying bug noticed is that it seems impossible to access the SD card.
Is this the case or did I miss something?

In the immediate future, any advice for a quality and non-intrusive android video converter, compatible with android 11, would be very useful to me.

Thanks in advance.
Best regards

**Comment by licaon-kter** [comment]
Share to the app does not work?

**Comment by gbdomubpkm** [comment]
Yes you right it works. But i speak of File search path in Settings. Even if i choose External storage, it doesn't work : choose a video always open on 
/storage/emulated/0.

**Comment by licaon-kter** [comment]
Yeah, unfortunate. Also it's a pain to use in general.

I find it easier to share from Gallery...

**Comment by gbdomubpkm** [comment]
Frankly, this app has everything to become excellent if some small improvements and updates could be done gradually (by priority).

**Comment by brarcher** [comment]
These days I don't have much time to contribute to the app, unfortunately. I think it would need an additional volunteer or so to carry it forward.

**Comment by gbdomubpkm** [comment]
I am not a developer so incompetent at this level, alas ! I understand very well your problems of time management.  I just find it unfortunate that this very useful app has been languishing for 3 years. Only a bug fixed every 2 months by any qualified developer  would fix things.

   Perhaps it is not well understood that a developer who wants to get involved is wanted.

   As it is, thanks for developing this app and hope it won't be broken even more by a future android update.

**Comment by brarcher** [comment]
> Perhaps it is not well understood that a developer who wants to get involved is wanted.

That's a good point. I've updated the README to make it clear that the app needs a volunteer to take over development:

https://github.com/brarcher/video-transcoder/blob/master/README.md#note-from-developer

> I just find it unfortunate that this very useful app has been languishing for 3 years

I'm glad the app has been useful.

**Comment by gbdomubpkm** [comment]
> I'm glad the app has been useful.

 IS always very useful  😉

**Comment by gbdomubpkm** [comment]
No more bugs found under android 12 AOSP: conversions can be done normally, or fail at startup, or are done partially. I'll try my luck and see what happens..  Maybe FFmpeg updates need to be done (audio and video formats).

**Comment by uhyf3** [comment]
I have Android 10 and Google Play won't let me install it.

**Comment by licaon-kter** [comment]
@uhyf3 still can from F-Droid ;)
